From: Alexander <sanek23994@gmail.com>
Date: Sat, 5 Jul 2014 22:13:12 +0200
Subject: clang FTBFS

Fix a FTBFS with the clang compiler due to uninitialized variables in
ttf2font.c.

Bug: https://bugs.debian.org/753064
Forwarded: no
---
 utils/ttf2font.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/utils/ttf2font.c b/utils/ttf2font.c
index 01ce00a..5300708 100644
--- a/utils/ttf2font.c
+++ b/utils/ttf2font.c
@@ -47,7 +47,8 @@ int main(int argc, char **argv)
 {
   FT_Library ftl;
   FT_Face face;
-  FT_UInt xsize, ysize;
+  FT_UInt xsize = 0;
+  FT_UInt ysize = 0;
   unsigned char ch;
   FILE *out = NULL;
 
